e2-studio-star-rx72n-firmware/libs/rx_hcsr04/src/rx_hcsr04.c

```
@@ -1564,22 +1673,79 @@ rx_err_t rx_hcsr04_init(rx_hcsr04_t* handle, const rx_hcsr04_config_t* config)
+      }
+    }
+
+    /* Configure pin for IRQ function via MPC (sets ISEL bit in PFS) */
+    err = rx_mpc_set_irq(config->echo_pin);
+    if (err != k_rx_ok) {
+      (void)hcsr04_hal_gpio_deinit(config->trigger_pin);
+      return err;
+    }
+
+    /* Determine IRQ priority: use configured value or fall back to default */
+    const uint8_t irq_priority = (config->irq_priority != (uint8_t)k_echo_irq_none)
```

Clang (14.0.6)

[warning] variable 'irq_priority' is not initialized

(cppcoreguidelines-init-variables)